src/main/java/storage/WriteDataFiles.java

```diff
@@ -85,16 +96,17 @@ private void buildFileContents(StringBuilder cheatSheetFileBuild, CheatSheet che
                 cheatSheet.getCheatSheetProgrammingLanguage());
         cheatSheetFileBuild.append(System.lineSeparator());
 
-        cheatSheetFileBuild.append(DETAILS).append(cheatSheet.getCheatSheetDetails());
+        cheatSheetFileBuild.append(DETAILS).append(WHITESPACE)
+                .append(cheatSheet.getCheatSheetDetails());
```

Comment:
I think can change to something like this for better readability
```
cheatSheetFileBuild
        .append(DETAILS)
        .append(WHITESPACE)
        .append(cheatSheet.getCheatSheetDetails());
```

Reply:
Noted! Thanks for highlighting